app/services/setting_registry.rb

```diff
@@ -0,0 +1,107 @@
+class SettingRegistry
+  include Singleton
+
+  def initialize
+    @settings = {}
```

Comment:
Just for my understanding (I like thinking in types) this is a `Hash[String, Setting]`, correct? I wonder if we should adopt yardoc style tags to have some documentation, but probably for another time.

Reply:
It is `Hash[String, SettingPresenter]`.
I agree with you there, but we should first talk about what to document and what style to use on discourse.

Reply:
Let's use this as input in that discussion.